frontend/src/components/BottomBanner/index.tsx

```diff
@@ -108,7 +108,7 @@ export default memo(function BottomBanner({
                     >
                       {NEWSLETTER_SIGNUP_BANNER_SUBSCRIBE_BUTTON_TEXT}
                     </StyledLink>
-                    {NEWSLETTER_SIGNUP_BANNER_SUBSCRIBE_TEXT}
+                    {NEWSLETTER_SIGNUP_BANNER_SUBSCRIBE_TEXT}{" "}
```

Comment:
What's the reason for adding whitespace here?

Reply:
Great question, to add space after the first period
<img width="119" alt="Screenshot 2024-10-28 at 1 28 17 PM" src="https://github.com/user-attachments/assets/1f4542b6-07c3-486b-b12e-41f7bb18cfcf">